client/state/stats/lists/utils.js

```diff
@@ -328,6 +328,50 @@ export const normalizers = {
 		return { page, pages, total, posts };
 	},
 
+	statsComments( data, query, siteId, site ) {
+		if ( ! data ) {
+			return null;
+		}
+		const adminUrl = site ? site.options.admin_url : null;
+
+		let authors = [];
+		if ( data.authors ) {
+			authors = data.authors.map( ( author ) => {
+				return {
+					label: author.name,
+					value: author.comments,
+					iconClassName: 'avatar-user',
+					icon: parseAvatar( author.gravatar ),
+					link: adminUrl + 'edit-comments.php' + author.link,
```

Comment:
Pending the choice on comment management in Calypso we will need to circle back to this link and update it eventually.